acquia_lift.module

```diff
@@ -39,3 +40,48 @@ function acquia_lift_page_attachments(array &$page) {
   // Attach lift CSS to all pages.
   $page['#attached']['library'][] = 'acquia_lift/acquia-lift';
 }
+
+/**
+ * Implements hook_toolbar().
+ */
+function acquia_lift_toolbar() {
+  $items = [];
+  $items['acquia_lift'] = [
+    '#cache' => [
+      'contexts' => [
+        'user.permissions',
+      ],
+    ],
+  ];
+
+  if (!\Drupal::currentUser()->hasPermission('access acquia lift links')) {
```

Comment:
Nice, you thought about access control and cacheability metadata! Woot! :)

Tip for making this simpler: remove lines 49 through 59, and instead add this to lines 61–84:

```
  '#access' => AccessResult::allowedIfHasPermission('access acquia lift links'),
```

That looks easier on the eyes, doesn't it? Declarative code FTW.